src/ui/public/filter_editor/filter_editor.html

```diff
@@ -0,0 +1,132 @@
+<div class="filterEditor kuiModal">
+  <div class="kuiModalHeader">
+    <div class="kuiModalHeader__title">
+      <span ng-show="filterEditor.filter.meta.isNew">Add</span>
+      <span ng-hide="filterEditor.filter.meta.isNew">Edit</span>
+      filter
+    </div>
+
+    <div
+      class="kuiModalHeaderCloseButton kuiIcon fa-times"
+      ng-click="filterEditor.onCancel()"
+    ></div>
+  </div>
+
+  <div class="kuiModalBody">
+    <div ng-show="filterEditor.showQueryDslEditor()">
+      <filter-query-dsl-editor
+        filter="filterEditor.filter"
+        on-change="filterEditor.setQueryDsl(queryDsl)"
+      ></filter-query-dsl-editor>
+      <div
+        ng-show="filterEditor.isEditingQueryDsl"
+        ng-click="filterEditor.isEditingQueryDsl = false"
+      >
+        <button class="kuiMenuButton kuiMenuButton--basic">
+          <span class="kuiMenuButton__icon kuiIcon fa-chevron-left"></span>
+          <span>Back</span>
+        </button>
+      </div>
+    </div>
+
+    <div
+      class="kuiFieldGroup kuiVerticalRhythm"
```

Comment:
If we add `kuiFieldGroup--alignTop` here, then each field will align to the top even if one of them is taller than the others.

![image](https://cloud.githubusercontent.com/assets/1238659/26739723/d2909338-4787-11e7-8b36-dc9df067bc48.png)